src/lib/utils/functions/EntityTypeUtils.ts

```diff
@@ -66,6 +66,26 @@ export function isContainerType(type: EntityType): boolean {
   }
 }
 
+export function isTableType(type: EntityType): boolean {
```

Comment:
Every time we add another function like this, I get a stronger urge to replace our data types with classes